services/github/pod-github/src/platform.ts

```diff
@@ -692,7 +692,7 @@ export class PlatformWorker {
         )
         let workspaceInfo: ClientWorkspaceInfo | undefined
         try {
-          workspaceInfo = await getWorkspaceInfo(token)
+          workspaceInfo = await getWorkspaceInfo(token, true)
```

Comment:
I’m concerned that abandoned workspaces with enabled integration might appear as active.

Reply:
They are active, since they have incoming changes from github.

Reply:
But now they are upgraded at the end.